internal/pkg/agent/cmd/install.go

```diff
@@ -67,6 +72,12 @@ func installCmd(streams *cli.IOStreams, cmd *cobra.Command) error {
 		return fmt.Errorf("unable to perform install command, not executed with %s permissions", utils.PermissionUser)
 	}
 
+	// only support Linux at the moment
+	nonRoot, _ := cmd.Flags().GetBool(flagInstallNonRoot)
+	if runtime.GOOS != "linux" {
```

Comment:
shouldn't this be `nonRoot && runtime.GOOS != "linux"`

Reply:
Yes it should. Fixed.